docs/developer-guide/working-with-rules.md

```diff
@@ -24,6 +24,9 @@ Here is the basic format of the source file for a rule:
 // Rule Definition
 //------------------------------------------------------------------------------
 
+/**
+ * @type {import('eslint').Rule.RuleModule}
```

Comment:
Doesn’t this require `@types/eslint`?

Reply:
Will discuss in this thread: https://github.com/eslint/generator-eslint/pull/113#discussion_r712613014